sharetribe/sharetribe
# Migrate from old session cookies to new

Prevents cookies from leaking to subdomains, and dynamically renames old cookies with
new key so that sessions are not invalidated after the change.

Removes old CustomDomainCookie and relies on Rails defaults.

Testing steps:

  1. Checkout master
  2. Run rails server
  3. Log in to some marketplace
  4. Checokut this branch and restart server
  5. Session should stay on, and visiting other subdomains shouldn't log you out, and the sessions should be separate.

Fixes #1192
